src/quo2/components/navigation/page_nav/style.cljs

```diff
@@ -0,0 +1,90 @@
+(ns quo2.components.navigation.page-nav.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(defn container
+  [margin-top]
+  {:margin-top         margin-top
+   :padding-horizontal 20
+   :padding-vertical   12
+   :height             56
+   :flex-direction     :row
+   :justify-content    :space-between
+   :align-items        :center})
+
+(defn center-content-container
+  [centered?]
+  {:flex              1
+   :margin-horizontal 12
+   :flex-direction    :row
+   :align-items       :center
+   :justify-content   (if centered? :center :flex-start)})
+
+(def account-switcher-placeholder
+  {:width            32
+   :height           32
+   :border-radius    10
+   :background-color (colors/custom-color :purple 50)})
+
+(def right-actions-container
+  {:flex-direction :row})
+
+(def right-actions-spacing
+  {:width 12})
+
+(def right-content-min-size
+  {:min-width 32 :min-height 32})
+
+(def token-logo
+  {:width 16 :height 16})
+
```

Comment:
I see a few styles in this file written on one line. Should the map be aligned properly?